.travis.yml

```diff
@@ -23,6 +23,16 @@ matrix:
         apt:
           sources: [ 'ubuntu-toolchain-r-test' ]
           packages: [ 'libstdc++6', 'libstdc++-5-dev' ]
+    # Clang format build
+    - os: linux
+      language: generic
+      env: BUILDTYPE=debug
+      node_js: 4
+      # Overrides `install` to avoid initializing clang toolchain
+      install: 
+        - ./scripts/format.sh
```

Comment:
Let's add `make format` target and call that here.

Reply:
Oh, also looks like you copied from `node-cpp-skel` matrix. So things are a little different here in how to override. Follow my example at https://github.com/mapbox/hpp-skel/pull/42/files#diff-354f30a63fb0907d4ad57269548329e3R5 as you just need to override `script` and you don't need to specify `node_js` or `env` or `language`.